src/utils/utils.tsx

```diff
@@ -72,6 +65,10 @@ interface ParentAssetObject extends ElementObject {
   comment: string;
   decommissioning_user?: string;
   time_decommissioned?: string;
+  cpu:string;
+  storage:string;
+  display_color:string;
+  memory_gb:string |null;
```

Comment:
Similar question about whitespace. Not a big deal tho, just curious if we need that